modules/n1ql_fts/pages/index.adoc

```diff
@@ -0,0 +1,589 @@
+
+
+== Prerequisites
+
+* Couchbase Server 6.5 Beta or Higher
+* Basic understanding of Couchbase and link:https://docs.couchbase.com/server/6.0/fts/full-text-intro.html[Full-Text Search]
+
+== Loading the Dataset
+
+For this demo we will use a dataset of movies exported from IMDB, execute the following steps to import it into your Couchbase instance:
+
+* Create a bucket called "movies"
+* Clone the *Couchflix* repository with the following command
+[source,]
+----
+git clone https://github.com/deniswsrosa/couchflix.git
+----
+    
+* Go to the *data* folder and extract the *datasets.zip* file
+* Run the following command to import the dataset into your *movies* bucket
+[source,]
+----
+./cbimport json -c couchbase://<IP_ADDRESS> -u <USERNAME> -p <PASSWORD> -b movies -d file://<PATH_TO_THE_FILE>/cb-movies-dataset2.json  -f list -g key::%id% -t 4
+----
+
+
+TIP: The *cbimport* location might vary according to where you have installed Couchbase. On *Mac* it is typically at /Applications/Couchbase Server.app/Contents/Resources/couchbase-core/bin
+
+* Go to the *indexes* folder and run the following command to create the index we are going to use.
+
+----
+curl -XPUT -H "Content-type:application/json" http://<USERNAME>:<PASSWORD>@<IP_ADDRESS>:8094/api/index/movies_shingle -d @movies_shingle.json
+----
+
+Finally, go to Couchbase's Web Console and click on the *Search* menu. If everything is right you should see something similar to the following:
+
+image:001-index-created.png[Index Created Successfully]
+
+
+== Searching for Movies
+
+=== Building a basic search
+
+Let's start by creating a query which searches for *Star War*. Note that we are searching for *Star War* instead of *Star WarS*. The goal is to build a simple search that already has some tolerance for typos.
+
+Run the following query via web console:
+
+[source, SQL]
+----
+
+select m.original_title, SEARCH_SCORE() AS score  from movies m where search(m, 
+    {
+       "should": { 
+                   "disjuncts": [
+                        {"field":"original_title", "match": "Star War"}, 
+                        {"field":"original_title", "match": "Star War", "fuzziness":1}
+                    ]
+             
+       }
+    }
+) 
+order by score desc
+limit 20
+
+----
+
+It will probably take a long time to run. You can see why in the query plan:
+
+image:002-slow-query-plan.png[Slow Query Plan]
+
+If you pay attention to the plan above, you will notice that no index has been used even though we have an index already in place. 
+
+Open the index we created via *Search -> movies_shingle*, you should see something similar to the following:
+
+image:003-index.png[movies index]
+
+Note that this definition only index documents with the type *_class* equals to *com.cb.fts.sample.entities.Movie*, and that is why our index wasn't used by our first query. If we rerun the same query filtering the result by the document type, it will much faster than before:
```

Comment:
- [ ] `only index documents` should be `only indexes documents`
- [ ] `equals to` should be `equal to`
- [ ] `it will much faster` should be `it will run much faster`